core/domain/exp_services.py

```diff
@@ -1943,11 +1955,12 @@ def revert_exploration(
     # change.
     exploration = exp_fetchers.get_exploration_by_id(
         exploration_id, version=revert_to_version)
-    exploration_rights = rights_manager.get_exploration_rights(exploration.id)  # type: ignore[no-untyped-call]
-    if exploration_rights.status != rights_domain.ACTIVITY_STATUS_PRIVATE:
-        exploration.validate(strict=True)  # type: ignore[no-untyped-call]
-    else:
-        exploration.validate()  # type: ignore[no-untyped-call]
+    exploration_rights = rights_manager.get_exploration_rights(exploration.id)
+    exploration.validate(
+        strict=(
+            exploration_rights.status != rights_domain.ACTIVITY_STATUS_PRIVATE
```

Comment:
Ditto.

Reply:
Done, the condition is pulled out into a new variable and passed that variable for strict.